nl_server/loader.py

```diff
@@ -46,18 +52,66 @@ def load_server_state(app: Any, embeddings_map: Dict[str, str],
     if nl_model and nl_embeddings:
       app.config[config.NL_MODEL_KEY] = nl_model
       app.config[config.NL_EMBEDDINGS_KEY] = nl_embeddings
+      app.config[config.NL_EMBEDDINGS_VERSION_KEY] = embeddings_map
       return
 
   nl_embeddings = embeddings_store.Store(config.load(embeddings_map,
                                                      models_map))
-  app.config[config.NL_EMBEDDINGS_KEY] = nl_embeddings
-
   nl_model = NLAttributeModel()
   app.config[config.NL_MODEL_KEY] = nl_model
+  app.config[config.NL_EMBEDDINGS_KEY] = nl_embeddings
+  app.config[config.NL_EMBEDDINGS_VERSION_KEY] = embeddings_map
 
   if _use_cache(flask_env):
     with Cache(cache.directory, size_limit=_NL_CACHE_SIZE_LIMIT) as reference:
       reference.set(NL_EMBEDDINGS_CACHE_KEY,
                     nl_embeddings,
                     expire=_NL_CACHE_EXPIRE)
       reference.set(NL_MODEL_CACHE_KEY, nl_model, expire=_NL_CACHE_EXPIRE)
+
+
+def _load_yamls(flask_env: str) -> tuple[Dict[str, str], Dict[str, str]]:
+  with open(get_env_path(flask_env, _MODEL_YAML)) as f:
+    models_map = yaml.full_load(f)
+  assert models_map, 'No models.yaml found!'
+
+  with open(get_env_path(flask_env, _EMBEDDINGS_YAML)) as f:
+    embeddings_map = yaml.full_load(f)
+  assert embeddings_map, 'No embeddings.yaml found!'
+
+  custom_map = _maybe_load_custom_dc_yaml()
+  if custom_map:
+    embeddings_map.update(custom_map)
+
+  return embeddings_map, models_map
+
+
+def _maybe_load_custom_dc_yaml():
+  base = os.environ.get('SQL_DATA_PATH')
+  if not base:
+    return None
+
+  file_path = os.path.join(base, f'data/nl/{_CUSTOM_EMBEDDINGS_YAML}')
```

Comment:
nit: Add a comment referencing the custom dc loader: https://github.com/datacommonsorg/website/blob/5ff56b7cdabb0c6c79e0c6c90bee8280c398ac2d/server/routes/admin/html.py#L40

Reply:
Done.